lib/recipient.rb

```diff
@@ -0,0 +1,55 @@
+require "httparty"
+require "awesome_print"
+require "dotenv"
+Dotenv.load
+
+class Recipient
+  attr_reader :is_user
+
+  def initialize(slack_id, is_user)
+    @slack_id = slack_id
+    @is_user = is_user
+  end
+
+  def self.details(slack_id)
```

Comment:
This method would make more sense as an instance method where you could get the details of that specific instance of `Recipient`.